src/js/tracing/timetodisplay.tsx

```diff
@@ -61,7 +61,9 @@ function TimeToDisplay(props: {
 
   if (__DEV__ && !nativeComponentMissingLogged && !nativeComponentExists) {
     nativeComponentMissingLogged = true;
-    logger.error('RNSentryOnDrawReporter is not available on the web, Expo Go and New Architecture. Run native build or report an issue at https://github.com/getsentry/sentry-react-native');
+    setTimeout(() => {
```

Comment:
Just a NIT so future contributors understand why we are using setTimeout here.

```suggestion
    // Using setTimeout with a delay of 0 milliseconds to defer execution and avoid printing the React stack trace.
    setTimeout(() => {
```